src/main/resources/application.properties

```diff
@@ -1,8 +1,9 @@
-
 management.endpoints.enabled-by-default=true
 management.endpoint.health.enabled=true
 management.endpoints.web.base-path=/officer-delta-processor
 management.endpoints.web.path-mapping.health=/healthCheck
 management.endpoint.health.show-details=ALWAYS
-
-server.port = 8080
\ No newline at end of file
+server.port=8080
+kafka.broker.url=${KAFKA_BROKER_URL:kafka:9092}
```

Comment:
do we need a chs-configs PR to go alongside these 3 kafka properties?
also in docker-chs branch delta-add-kafdrop -  we have `KAFKA_BROKER_ADD`
and the polling items are missing

Reply:
Yes we do need a config PR. I was just relying on the defaults for missing vars, but being explicit, and having the variables in the properties file would be better. I'll make that change now.

Reply:
Config PR: https://github.com/companieshouse/chs-configs/pull/1614